pluginlib/include/pluginlib/class_loader.hpp

```diff
@@ -90,7 +90,7 @@ class ClassLoader : public ClassLoaderBase
    * \return An instance of the class
    * \deprecated use either createInstance() or createUnmanagedInstance()
    */
-  __attribute__((deprecated)) T * createClassInstance(
+  ROS_DEPRECATED T * createClassInstance(
```

Comment:
would `[[deprecated]]` work for your use case? this would allow to stay closer to the `ros2` branch

Reply:
the compiler that I use support[[deprecated]] but I'm not sure if other compilers all support C++14.  If all compilers used to build ROS support it, I can change it to [[deprecated]], please advise.

Reply:
ROS Melodic assumes C++14 compatible compilers (see [REP3](http://www.ros.org/reps/rep-0003.html#melodic)), so feel free to go ahead and switch to `[[deprecated]]`

Reply:
cool, change to use standard attribute